package/src/components/AccountProfileInfo/v1/AccountProfileInfo.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { withComponents } from "@reactioncommerce/components-context";
+import { addTypographyStyles, applyTheme } from "../../../utils";
+
+const AccountProfileInfoContainer = styled.div`
+  display: flex;
+  position: relative;
+`;
+
+const AccountProfileInfoTextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+`;
+
+const ViewerEmailText = styled.span`
+  ${addTypographyStyles("ViewerInfoInitials", "labelText")}
+  color: ${applyTheme("accountProfileInfoEmailFontColor")};
+  font-size: ${applyTheme("accountProfileInfoEmailFontSize")};
+  align-self: left;
+  margin-bottom: 4px;
+  margin-left: 16px;
+`;
+
+const ViewerNameText = styled.span`
+  ${addTypographyStyles("ViewerInfoInitials", "labelText")}
+  font-size: ${applyTheme("accountProfileInfoNameFontSize")};
```

Comment:
Change `ViewerInfoInitials` to `AccountProfileInfoName` and remove `font-size`. Looking at the designs, I think just changing "labelText" to "titleTextBold" will remove the need for overriding font-size.